renderdoc/replay/replay_driver.h

```diff
@@ -37,6 +37,26 @@ struct FetchFrameRecord
   rdctype::array<FetchDrawcall> drawcallList;
 };
 
+struct GetTextureDataParams
+{
+  bool forDiskSave;
+  FormatComponentType typeHint;
+  bool resolve;
+  bool forceRGBA8unorm;
```

Comment:
I think what I'd suggest is changing this from `forceRGBA8unorm` to an enum that has a handful of values - one for "don't remap" and then one each for the desired destination formats, e.g. `RGBA8`, `RGBA16`, `RGBA32`, `D32S8` or whatever is necessary.